tests/integration/utils.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# SPDX-License-Identifier: MIT
+
+import requests
+from requests_kerberos import HTTPKerberosAuth
+
+
+class Response:
+    """Contains simplified HTTP response
+
+    :attribute dict data: Data fetched from the Cachito API
+    :attribute int id: ID of the request in Cachito
+    :attribute int status: HTTP status code
+    """
+    def __init__(self, data, id, status):
+        self.data = data
+        self.id = id
+        self.status = status
+
+
+class Client:
+    """Class that simplifies access to the Cachito API
+
+    :attribute string _cachito_api_url: URL of the Cachito API
+    """
+    def __init__(self, cachito_api_url):
+        self._cachito_api_url = cachito_api_url
+
+    def fetch_all_requests(self):
+        """Fetch a list of requests from the Cachito API
+
+        :return: Object that contains response from Cachito API
+        :rtype: Response
+        """
+        resp = requests.get(f"{self._cachito_api_url}requests")
+        resp.raise_for_status()
+        return Response(resp.json(), None, resp.status_code)
+
+    def fetch_request(self, request_id):
+        """Fetch a request from Cachito API
+
+        :param request_id: ID of the request in Cachito
+        :return: Object that contains response from Cachito API
+        :rtype: Response
+        """
+        resp = requests.get(f"{self._cachito_api_url}requests/{request_id}")
+        return Response(resp.json(), resp.json()["id"], resp.status_code)
+
+    def create_new_request(self, **kwargs):
```

Comment:
Suggestion: use `payload` instead of `**kwargs`. That way if we do need to pass in another parameter to this method, it doesn't end up in the request payload.

Reply:
changed